README.md

```diff
@@ -246,26 +246,26 @@ The following environment variables must be set when deploying the app.
 
 | Variable Name                             | Description                                                                          |
 |-------------------------------------------|--------------------------------------------------------------------------------------|
-| SUBMISSION_BUCKET_NAME                    | The name of the bucket that submissions will be stored in                            |
+| EQ_SUBMISSION_BUCKET_NAME                 | The name of the bucket that submissions will be stored in                            |
```

Comment:
This should be removed, it is optional now.

Reply:
Removed it from here, thanks!